docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Do we need this file? Usually `.bat` files are only for running on Window OSs.

Reply:
Sphinx automatically generated it. We probably don't need it. I'll remove it in the next iteration.